### PR TITLE
Added the dockerfile to run the bridge

### PR DESCRIPTION
**Pull request by AntoineRondelet**
See issue #5

### CODE REVIEW

File: wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+node usr/azure-integration/examples/simple.js
```
Comment (johanstokking) [inline wrapper.sh]: Why running the simple example?

File: wrapper.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+printf 'INTEGRATION PROCESS FATAL ERROR: %s\n' "$?"
+
+exit 1
```
Comment (johanstokking) [inline wrapper.sh]: Why always exiting with 1?

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: server.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+var ttn = require('ttn');
+
+var appId = 'office-app';
+var accessKey = 'ttn-account-preview.OfuuW9smtu33PjpPtVAs54Bmc2dcgHEOywtuAT1oqzk';
+var region = 'eu'
```
Comment (johanstokking) [inline server.js]: Read these from the environment

File: server.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+client.on('connect', function(connack) {
+  console.info('[INFO]', 'Connected: ' + JSON.stringify(connack, null, 2));
+});
+
+client.on('error', function(err) {
+  console.error('[ERROR]', err.message);
+});
+
+client.on('activation', function(e) {
+  console.log('[INFO] ', 'Activated: ', e.devEUI);
```
Comment (johanstokking) [inline server.js]: No need to add spaces; `console.log()` does that for you

File: server.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+});
+
+client.on('message', function(msg) {
+  console.info('[INFO] ', 'Uplink: ' + JSON.stringify(msg, null, 2));
```
Comment (johanstokking) [inline server.js]: Be consistent with passing fields by argument or as concatenated strings

File: wrapper.sh [hunk 6]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-node usr/azure-integration/examples/simple.js
+node usr/azure-integration/bridge.js
 printf 'INTEGRATION PROCESS FATAL ERROR: %s\n' "$?"
```
Comment (johanstokking) [inline wrapper.sh]: Print a fatal error when `node` crashes; check its exit status

**Review by johanstokking — CHANGES_REQUESTED** [review]

File: wrapper.sh (hunk 6 above)
Comment (AntoineRondelet) [reply wrapper.sh]: Isn't it already the case @johanstokking ? the bridge is an infinite loop. The only way to reach the print is when a hard exit happens which makes us exit the bridge program. Then I print the fatal error message and print the status

**Comment by johanstokking** [comment]
Awesome. How big becomes this image in MB?

**Comment by AntoineRondelet** [comment]
50 MB @johanstokking

File: wrapper.sh [hunk 7]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+node usr/azure-integration/bridge.js
+printf 'INTEGRATION PROCESS FATAL ERROR: %s\n' "$?"
+
+exit 0
```
Comment (johanstokking) [inline wrapper.sh]: Return the exit code here. I now see what this wrapper is for. It should only print the fatal error when the exit code != 0.

On a graceful shutdown (SIGTERM), the process should shutdown with exit 0 and this is not a fatal error. When updating or scaling down, an exit 0 is fine and should not initiate a crash report.

**Review by johanstokking — CHANGES_REQUESTED** [review]

**Review by johanstokking — APPROVED** [review]

**Pull request merged**